servicenow/stackstate_checks/servicenow/data/conf.yaml.example

```diff
@@ -7,8 +7,14 @@ instances:
     user: admin
     password: Service@123
 
-    # Batch size for paginating results. Default value is 2500
-    # batch_size: 1000
+    # Batch size for paginating results. Default value is 2500. Max value is 10000.
+    # batch_size: 2500
+
+    # On first start we get all changed request that have been updated in last N days. Default value is 100.
+    # crs_updated_in_days: 30
```

Comment:
we should change the name to something meaningful like `change_request_bootstrap_days` following the same structure as dynatrace. so everybody is on same page.

Reply:
Done.